.github/workflows/ci.yml

```diff
@@ -0,0 +1,48 @@
+name: Python CI Workflow
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+    
+
+jobs:
+  test:
+    name: GHS Test
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version:
+          - "3.7"
+          - "3.8"
+          - "3.9"
+          - "3.10"
+
+    steps:
+      - name: Checkout infrastructure
+        uses: actions/checkout@v3
+
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Upgrade pip version
+        run: |
+          python3 -m pip install --upgrade pip
+
+      - name: Install the project
+        run: |
+          pip install .
```

Comment:
Just to be consistent with other commands, can we change this to `pip3`?

Reply:
Done!